src/components/AnnouncementDate/AnnouncementDate.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import styles from './AnnouncementDate.module.scss'
+
+type PropTypes = {
+  date: string
+}
+
+const AnnouncementDate = ({ date }: PropTypes) => {
```

Comment:
nit-pick (non-blocking): This component isn't complex, but might be nice to add a test for it.

Reply:
I thought the same, but after running `yarn test --coverage` it showed that it had 100% coverage. I'm assuming because it's used in a larger test file? I'll go ahead and add a test though.